src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -2610,35 +2633,96 @@ .locals init (delegate*<string,object> V_0)
 ");
         }
 
+        [Theory]
+        [InlineData("in")]
+        [InlineData("ref")]
+        public void AddressOf_Initializer_Overloads(string refType)
```

Comment:
>AddressOf_Initializer_Overloads [](start = 20, length = 31)

Do we have an error test for ref/out mismatch?

Reply:
This was supposed to be `FunctionPointerToFunctionPointerInvalid_ParameterVariance`

---
In reply to: [409817853](https://github.com/dotnet/roslyn/pull/43355#discussion_r409817853) [](ancestors = 409817853)

Reply:
But I'll add a method group resolution test as well.

---
In reply to: [410493966](https://github.com/dotnet/roslyn/pull/43355#discussion_r410493966) [](ancestors = 410493966,409817853)

Reply:
Additional cases added to `AddressOf_RefsMustMatch`.

---
In reply to: [410494067](https://github.com/dotnet/roslyn/pull/43355#discussion_r410494067) [](ancestors = 410494067,410493966,409817853)